src/search/executors/filter_executor.h

```diff
@@ -100,6 +100,9 @@ struct QueryExprEvaluator {
         return l > r;
       case NumericCompareExpr::GET:
         return l >= r;
+      default:
+        assert(false);
+        return false;
```

Comment:
Should we elaborate all branches rather than a unreachable "default"

Reply:
Actually all branches are covered here, but gcc thought it should have a default branch to cover any unforeseen case.